packages/@uppy/locales/src/de_DE.js

```diff
@@ -1,150 +1,184 @@
-  failedToUpload: 'Fehler beim Upload von Datei %{file}',
-  fileSource: 'Dateiquelle: %{name}',
+  failedToUpload: 'Fehler beim Hochladen der Datei %{file}',
   filesUploadedOfTotal: {
     '0': '%{complete} von %{smart_count} Datei hochgeladen',
     '1': '%{complete} von %{smart_count} Dateien hochgeladen',
-    '2': '%{complete} von %{smart_count} Dateien hochgeladen',
   },
   filter: 'Filter',
-  finishEditingFile: 'Dateibearbeitung beenden',
+  finishEditingFile: 'Bearbeitung beenden',
+  flipHorizontal: 'Horizontal spiegeln',
   folderAdded: {
-    '0': '%{smart_count} Datei von %{folder} hinzugefügt',
+    '0': 'Eine Datei von %{folder} hinzugefügt',
     '1': '%{smart_count} Dateien von %{folder} hinzugefügt',
-    '2': '%{smart_count} Dateien von %{folder} hinzugefügt',
   },
-  import: 'Import',
+  folderAlreadyAdded: 'Der Ordner "%{folder}" wurde bereits hinzugefügt',
+  generatingThumbnails: 'Erstellen von Miniaturansichten...',
+  import: 'Importieren',
+  importFiles: 'Importiere Dateien von:',
   importFrom: 'Importieren von %{name}',
+  inferiorSize: 'Diese Datei ist kleiner als die minimal erlaubte Dateigröße von %{size}',
   loading: 'Laden...',
-  logOut: 'Ausloggen',
+  logOut: 'Abmelden',
+  micDisabled: 'Zugriff auf Mikrofon von Benutzer abgelehnt',
+  missingRequiredMetaField: 'Fehlende erforderliche Meta-Felder',
+  missingRequiredMetaFieldOnFile: 'Fehlende erforderliche Meta-Felder in %{fileName}',
   myDevice: 'Mein Gerät',
   noCameraDescription: 'Bitte Kamera anschließen, um Bilder oder Videos aufzunehmen',
   noCameraTitle: 'Kamera nicht verfügbar',
   noDuplicates: 'Datei \'%{fileName}\' existiert bereits und kann nicht erneut hinzugefügt werden',
-  noFilesFound: 'Sie haben keine Dateien oder Ordner hier',
+  noFilesFound: 'Sie haben hier keine Dateien oder Ordner',
   noInternetConnection: 'Keine Internetverbindung',
   noNewAlreadyUploading: 'Während der Upload läuft, können keine weiteren Dateien hinzugefügt werden',
+  openFolderNamed: 'Ordner %{name} öffnen',
   pause: 'Pausieren',
-  pauseUpload: 'Upload pausieren',
   paused: 'Pausiert',
-  poweredBy: 'Angetrieben von %{uppy}',
+  pauseUpload: 'Hochladen pausieren',
+  pluginNameBox: 'Box',
+  pluginNameCamera: 'Kamera',
+  pluginNameDropbox: 'Dropbox',
+  pluginNameFacebook: 'Facebook',
+  pluginNameGoogleDrive: 'Google Drive',
+  pluginNameInstagram: 'Instagram',
+  pluginNameOneDrive: 'OneDrive',
+  pluginNameZoom: 'Zoom',
+  poweredBy: 'Powered by %{uppy}',
   processingXFiles: {
-    '0': '%{smart_count} Datei verarbeiten',
+    '0': 'Eine Datei verarbeiten',
     '1': '%{smart_count} Dateien verarbeiten',
-    '2': '%{smart_count} Dateien verarbeiten',
   },
+  recording: 'Aufnahme',
   recordingLength: 'Aufnahmedauer %{recording_length}',
+  recordingStoppedMaxSize: 'Die Aufnahme wurde gestoppt, weil die Dateigröße das Limit überschreitet',
```

Comment:
```suggestion
  recordingStoppedMaxSize: 'Die Aufnahme wurde gestoppt, weil die Dateigröße das Limit überschritten hat',
```

I think this makes more sense, time wise

Reply:
You're right, thanks!